dev/tests/js/framework/JsTestRunner.php

```diff
@@ -0,0 +1,537 @@
+<?php
+
+class JsTestRunner
```

Comment:
How is this class more maintainable? I agree with some refactorings but what's the benefit of having functions namespaced in class?